OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 16 Pro.json

```
@@ -18,9 +18,8 @@
   "DigitizerIdentifiers": [
     {
       "VendorID": 10429,
-      "ProductID": 2379,
-      "InputReportLength": 12,
-      "OutputReportLength": 12,
+      "ProductID": 36874,
+      "InputReportLength": 10,
```

Based on the latest diagnostics file you posted:
```suggestion
      "InputReportLength": 10,
      "OutputReportLength": 0,
```

Please double test that this works - if it doesn't, the `OutputReportLength` needs to be `10` instead.

---

`"OutputReportLength": 10` did the trick where 0 didn't